hw/ip/prim/dv/prim_present/crypto_dpi_present/Makefile

```diff
@@ -0,0 +1,17 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+.DEFAULT_GOAL := all
+
+all: build
+
+###################
+## build targets ##
+###################
+
+build:
```

Comment:
suggestion: 

call this target `crypto_present.so`. add `crypto_present.c` as a dependency to this target, and add this target as a dependency to `all`. remove the `.PHONY` line below.

and add a `clean` target that does `rm -rf crypto_present.so`.

Reply:
Thank you for the detailed suggestion Michael! Yeah that makes it more clear than the generic namings.